tests/headers/host_test.go

```diff
@@ -111,3 +104,53 @@ func TestHostHeaderMultiple(t *testing.T) {
 		t.Errorf("status code got: %q want: %q", got, want)
 	}
 }
+
+func TestAbsoluteFormURLInvalidSchema(t *testing.T) {
+	// (I (grenfeldt@) don't actually know whether this
```

Comment:
I'd leave out this part and maybe open an issue if you think it's something we should investigate :)